core/client/fs/src/main/java/alluxio/client/block/stream/GrpcBlockingStream.java

```diff
@@ -67,12 +68,14 @@
   /**
    * @param rpcFunc the gRPC bi-directional stream stub function
    * @param bufferSize maximum number of incoming messages the buffer can hold
+   * @param description description of this stream
    */
   public GrpcBlockingStream(Function<StreamObserver<ResT>, StreamObserver<ReqT>> rpcFunc,
-      int bufferSize) {
+      int bufferSize, String description) {
     mResponses = new ArrayBlockingQueue<>(bufferSize);
     mResponseObserver = new ResponseStreamObserver();
     mRequestObserver = (ClientCallStreamObserver) rpcFunc.apply(mResponseObserver);
+    mDescription = description;
```

Comment:
Does this need to be passed in or can we derive it from some state the stream already has?

Reply:
In this case we want to include the worker address which is not exposed from the stream.